src/components/image-plus.js

```diff
@@ -180,6 +144,7 @@ AFRAME.registerComponent("image-plus", {
       texture.minFilter = THREE.LinearFilter;
       videoEl.addEventListener("loadedmetadata", () => resolve(texture), { once: true });
       videoEl.onerror = reject;
+      texture.audioSource = this.el.sceneEl.audioListener.context.createMediaElementSource(videoEl);
```

Comment:
We shouldn't attach arbitrary objects to existing three.js constructs. If you need to associate the audioSource with a texture you should create a map in this component.